src/interpreter/kernel/instruction/binop.js

```diff
@@ -58,12 +59,41 @@ function binop(
   throw new Error('Unsupported binop: ' + sign);
 }
 
+function binopi64s(
+  c1: StackLocal,
+  c2: StackLocal,
+  sign: Sign,
+  createValue: (Long) => StackLocal
+): StackLocal {
+  
+  let res;
+
+  switch (sign) {
+  // https://webassembly.github.io/spec/exec/numerics.html#op-iadd
```

Comment:
Btw this link changed: https://webassembly.github.io/spec/core/exec/numerics.html#op-iadd